src/schema/feeds.ts

```diff
@@ -1171,6 +1208,61 @@ export const resolvers: IResolvers<any, Context> = traceResolvers({
       }
       return feedResolverV1(source, args, ctx, info);
     },
+    customFeed: async (
+      source,
+      args: ConfiguredFeedArgs,
+      ctx: Context,
+      info,
+    ) => {
+      const feedIdOrSlug = args.feedId;
```

Comment:
Oh configuredFeedArgs already has it, so we can remove it in above query as extra?